test/fixtures/node_pool/terraform.tfvars.sample

```diff
@@ -0,0 +1,8 @@
+project_id=""
+credentials_path="../../../credentials.json"
```

Comment:
This path appears to be relative to `test/fixtures/node_pool`, but the other examples use `../..` which I think is relative to `examples/{example}`. Can we fix this up?